e2e-tests/cypress/integration/appeal-submission-site-health-and-safety-issues.feature

```diff
@@ -0,0 +1,60 @@
+@wip
+Feature: Appellant submission - Health and safety issues
+  As an Inspector I need to understand if there are any H&S concerns at the appeal site so that I can plan
+  my site visit accordingly
+
+ # On page load none of the option should be selected, this will include the page navigation from task list
```

Comment:
This one means that when we just click on "Yes" there is an input popping up ?

Reply:
Otherwise I don't see the difference between
   Given the status of the appeal section is displayed
and
   When health and safety issues is accessed

Reply:
added new scenarios after discussing with Frank